[2024-01-31T12:29:25.689+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-23T00:00:00+00:00 [queued]>
[2024-01-31T12:29:26.398+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-23T00:00:00+00:00 [queued]>
[2024-01-31T12:29:26.402+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-01-31T12:29:26.655+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_hello_task> on 2024-01-23 00:00:00+00:00
[2024-01-31T12:29:26.712+0000] {standard_task_runner.py:60} INFO - Started process 132 to run task
[2024-01-31T12:29:26.764+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hola_mundo', 'print_hello_task', 'scheduled__2024-01-23T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/prueba.py', '--cfg-path', '/tmp/tmpkkgr25rm']
[2024-01-31T12:29:26.811+0000] {standard_task_runner.py:88} INFO - Job 27: Subtask print_hello_task
[2024-01-31T12:29:27.970+0000] {task_command.py:423} INFO - Running <TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-23T00:00:00+00:00 [running]> on host 60382a02b079
[2024-01-31T12:29:30.107+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='usuario' AIRFLOW_CTX_DAG_ID='hola_mundo' AIRFLOW_CTX_TASK_ID='print_hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-23T00:00:00+00:00'
[2024-01-31T12:29:30.139+0000] {logging_mixin.py:188} INFO - Hola Mundo!
[2024-01-31T12:29:30.154+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T12:29:30.285+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hola_mundo, task_id=print_hello_task, execution_date=20240123T000000, start_date=20240131T122925, end_date=20240131T122930
[2024-01-31T12:29:30.478+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T12:29:35.515+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
